fontforgeexe/charview.c

```diff
@@ -3469,8 +3469,13 @@ return( icon );
 
 static int CVCurEnc(CharView *cv)
 {
-    if ( cv->map_of_enc == ((FontView *) (cv->b.fv))->b.map && cv->enc!=-1 )
-        return( cv->enc );
+/* This is because cv->enc is not updated when FontForge thinks multiple
+ * glyphs are open, which is almost always. See the condition in
+ * _CVMenuChangeChar (testing GGadgetGetTitle8), which triggers whenever
+ * squared brackets are displayed in the word list ("[x]").
+ */
+//    if ( cv->map_of_enc == ((FontView *) (cv->b.fv))->b.map && cv->enc!=-1 )
+//        return( cv->enc );
```

Comment:
I'm not sure why we need comments for code you _disabled_ as if it would in some other context still be valid to re-enable…certainly the commit/PR notes would suffice for this?

Reply:
I guess I was paranoid, sorry.